specs/fast-project-realignment/README.md

```diff
@@ -0,0 +1,223 @@
+
+#### Migrate to the Popover API
+
+Leverage of the Foundation anchored region is one of the largest and most critical polyfills in Nimble powering all popovers (select, combobox, rich text mentions, tooltip). We should align with the native Popover API which provides new capabilities that cannot be given consistently with the anchored region polyfill (the ability to place content in top-layer).
+
+#### Migrate off of Foundation Design Tokens
+
+The FAST Foundation Design Token infrastructure has thus far been fairly complex, a source of performance issues, and is no longer maintained upstream. The FluentUI library has completely removed the Design Token infrastructure.
+
+There are several major ways we leverage FAST Foundation Design Tokens:
+- As CSS custom properties coupled to a theme token.
+- As non-CSS reflected custom properties for theme and localization tokens.
+- As non-CSS reflected custom properties for native attirbutes of lang and dir.
+
+Some research areas (likely more than one needed to fully replace and not comprehensive):
+- See how [FluentUI migrated away from Design Tokens](https://github.com/microsoft/fluentui/pull/30002).
+
+    Notes: Having token variable names shared in CSS seems reasonable, but solution lacks programmatic observability and seems to pollute inline styles on body tag / sprouts inline styles on user-owned elements.
+- See Lit documentation of [Community Context Protocol](https://lit.dev/docs/data/context/) which cites themes and localization as potential use-cases.
+
+    Notes: Likely a useful part of the solution. Is an open protocol that could potentially be used across frameworks / libraries but would need to validate. FAST had an implementation that added [context protocol support to the Foundation DI system in the Next packages](https://github.com/microsoft/fast/pull/6053).
+- See [`StyleObserver`](https://github.com/bramus/style-observer) library which enables programmatically observability of CSS custom property changes.
+
+    Notes: Potentially an interesting way to add observability to CSS custom property changes. Our Design Tokens that need observation are generally not reflected as CSS custom properties today (theme, localization, lang, dir) so we could make them CSS custom properties to leverage the pattern or replace them with an implementation on Community Context Protocol.
+
+#### Leverage heterogeneous base classes
+
+FAST is a relatively small community that no longer has a large shared resource of component development. Web components as a technology do not require us to ship only components built on FAST base classes. We should research utilitization of alternative base classes, such as Lit Element, being shipped with Nimble to give access to a larger community of shared component implementations.
+
+## Open issues
+
+- Should we publicize the fork in the FAST community / discord?
```

Comment:
I don't think that answering this needs to block completion of the HLD / feature, we can decide it once the fork is set up and published. 

My take right now: I'm not opposed to publicizing it. But the questions I would ask as a potential external consumer are things like "what is your maintenance plan?", "how much new investment will this receive?", and "will you accept my contributions?". While we have hopeful answers to those, we might want to establish some kind of track record (e.g. a couple months of Renovate updates or bug fixes) before we convince anyone to use this.

Reply:
Agreed, and I think the "Update top-level README / CONTRIBUTING to cover intentions of the repo / fork:" section describes high-level guidelines that should answer those questions and we can get the language precise on update and decide whether to share in discord after "some time" in production.